test/sync/test_base.py

```diff
@@ -0,0 +1,36 @@
+######################################################################
+#
+# File: test/sync/test_base.py
+#
+# Copyright 2020 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from contextlib import contextmanager
+import re
+import unittest
+
+
+class TestBase(unittest.TestCase):
```

Comment:
isn't this file a copy of another file? Could it not have been imported?

Reply:
When working on a new project, I follow the existing conventions.  In this case, I followed the existing convention of the other three existing `test_base.py` files.

I think we're veering off the point of this PR, which was to prevent the directory travesal attack, not refactor all the existing code.

Reply:
I'll file a separate issue for it

Reply:
https://github.com/Backblaze/b2-sdk-python/issues/131